Reference/conformance/conformance.pb.swift

```diff
@@ -265,7 +265,7 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message, SwiftProtobuf.Prot
       self = .None
     }
 
-    public mutating func decodeField(setter: inout SwiftProtobuf.FieldDecoder, protoFieldNumber: Int) throws {
+    public mutating func decodeField<T: SwiftProtobuf.FieldDecoder>(setter: inout T, protoFieldNumber: Int) throws {
```

Comment:
At first I thought you might have added an associated type or `Self` requirement to `FieldDecoder` that would prompt this change, but I don't see that. Is it a performance boost because it causes the compiler to specialize implementations instead of using dynamic dispatch? If so, is the size impact on the runtime and generated code minimal?